README.md

```diff
@@ -1,5 +1,375 @@
-# sumo_rest
-Generic cowboy handlers to work with Sumo
+# Sumo Rest
+
+<img src="http://www.technovelgy.com/graphics/content/sumo_robot.jpg" align="right" style="float:right" height="400" />
+
+Generic cowboy handlers to work with Sumo D
+
+## Introduction
+We, at Inaka, are used to build our RESTful servers on top of [cowboy](https://github.com/ninenines/cowboy). We use [sumo_db](https://github.com/inaka/sumo_db) to manage our persistence and [trails](https://github.com/inaka/cowboy-trails) together with [cowboy-swagger](https://github.com/inaka/cowboy-swagger) for documentation.
```

Comment:
Minor nitpick, "used to" denotes habit, as in we build on top of cowboy because that's what we are comfortable with, instead of because it is the best technical option :P